includes/class-freemius.php

```diff
@@ -9862,37 +9934,29 @@ private function send_install_update( $override = array(), $flush = false ) {
          *
          * @param string[] string $override
          * @param bool     $flush
+         * @param bool     $is_two_way_sync @since 2.5.0 If true and there's a successful API request, the install sync cron will be cleared.
          *
          * @return false|object|string
          */
-        private function send_installs_update( $override = array(), $flush = false ) {
+        private function send_installs_update( $override = array(), $flush = false, $is_two_way_sync = false ) {
             $this->_logger->entrance();
 
-            $installs_data = $this->get_installs_data_for_api( $override, ! $flush );
+            /**
+             * Pass `true` to use the network level storage since the update is for many installs.
+             *
+             * @author Leo Fajardo (@leorw)
+             * @since 2.2.3
+             */
+            $keepalive_only_update = $this->should_send_keepalive_update( true );

```

Comment:
@fajardoleo the purpose/meaning of this param has changed now and it is no longer indicating whether it's only a keepalive update or not, so I propose to refactor to `$should_send_keepalive`.

Reply:
@vovafeldman Renamed, thanks.